db.py

```diff
@@ -0,0 +1,145 @@
+from sqlalchemy import Column, Integer, Float, String, Boolean
+from sqlalchemy import create_engine
+
+Base = declarative_base() #Make it an OOP
+engine = create_engine('postgresql://@localhost:5432/myDB', echo = True)
+
+class Account(Base):
+    """
+    The main user class for ReeDistribute Login.abs
+    Attributes: account_id (int), name (str), email (str), consumer_key (str), anon (int), admin (int)
+    """
+    __tablename__='accounts' #Store all account objects in accounts
+
+    account_id = Column(Integer, primary_key=True, nullable = False) #The key value for account account_identification
+    username = Column(String, nullable = False) #Account's chosen name, used for site personalization
+    email = Column(String, nullable = False) #Account's email for contacting
+    consumer_key = Column(String) #Account's Stripe coustomer key
+    anon = Column(Integer) #If 0 then the account does not wish to remain anonymous for actions on the website
+    admin = Column(Integer) #If 0 then this account is not an admin
+    def __init__(self, account_id, username, email, consumer_key, anon = 1, admin = 0):
+        # Initialize account to the input values
+        self.account_id = account_id
+        self.username = username
+        self.consumer_key = consumer_key 
+        self.anon = anon
+        self.admin = admin
+
+    def __repr__(self):
+        # When called, accounts return the user's name
+        return self.username
+
+class Student(Base):
+    """
+    The class for LSES students
+    Attributes: account_id, open_requests (int), closed_requests (int), gained_money (int)
+    """
+    __tablename__ = 'students' #store all student Objects in students
+
+    account_id = Column(Integer, primary_key=True, nullable = False) #The key value for account identification
+    open_requests = Column(Integer) #The number of open requests by the student
+    closed_requests = Column(Integer) #The number of requests which the student opened that are now closed (filled)
+    gained_money = Column(Float) #The total amount of money this Student has gained (used for statistics)
+
+    def __init__(self, account_id):
+        # Initializes the student with the account id and all other attributes set to 0
+        self.account_id = account_id
+        self.open_requests = 0
+        self.closed_requests = 0
+        self.gained_money = 0
+    def __repr__(self):
+        # Formats the student's information to be easily read
+        s = "<ID: %d, Opened Requests: %d, Closed Requests: %d, Money Gained: %d>" % (self. account_id, self.open_requests, self.closed_requests, self.gained_money)
+        return s
+
+class Donor(Base):
+    """
+    The class for Donors
+    Attributes:  account_id (int), closed_requests (Int), money_given (int)
+    """
+    __tablename__= 'donors' #Store all donor objects in the table donors
+
+    account_id = Column(Integer, primary_key=True, nullable = False) #The key value for account identification

```

Comment:
No, I was creating accounts for donors so we can record the statistics R4R suggested.